pennylane/qnodes/base.py

```diff
@@ -199,15 +195,16 @@ def __init__(self, func, device, *, mutable=True, properties=None):
         self.output_dim = None  #: int: dimension of the QNode output vector
         self.model = self.device.capabilities()["model"]  #: str: circuit type, in {'cv', 'qubit'}
 
-    def __str__(self):
-        """String representation"""
+    def __repr__(self):
+        """String representation."""
         detail = "<QNode: device='{}', func={}, wires={}>"
         return detail.format(self.device.short_name, self.func.__name__, self.num_wires)
 
-    __repr__ = __str__
-
     def print_applied(self):
-        """Prints the most recently applied operations from the QNode."""
+        """Prints the most recently applied operations from the QNode.
+
+        FIXME what's the purpose of this? Could be a CircuitGraph method.
+        """
```

Comment:
This was introduced some while ago as a means for people to easily access what gates have been applied to the qnode. I guess you're right about making this a circuit_graph method (like draw for circuit diagrams). But it should still be accessible from the qnode

Reply:
Done.